case/Input/SickleCell.dat

```diff
@@ -0,0 +1,4 @@
+          12          24
```

Comment:
this looks like nlat and nlon and a dealiasing factor.

Reply:
Yep, this was the nlat and nlon (12, 24) of the cell in the file. So, when I import the cell, it will always be with these dimensions (defined in the file when exported).